### PR TITLE
Update bgtask sample

### PR DESCRIPTION
**Pull request by angelazhangmsft**

### CODE REVIEW

File: src/Samples/BgTaskComponent/README.md [hunk 1]
```diff
@@ -2,19 +2,51 @@
...
+
+### Deploy and run the  sample
+
+1. Select **Build** -> **Deploy Solution** to deploy the sample.
+
+2. To debug and run the sample, press F5 or select **Debug** -> **Start Debugging**.
+
+### Triggering background tasks
+
+The **ToastBgTask** background task raises a toast notification and is triggered by a TimeZoneChange event. You can invoke the background task while running the sample either by using the **Lifecycle Events** dropdown in Visual Studio, or by changing your device's timezone in **Date & time settings**.
```
Comment (j0shuams) [inline src/Samples/BgTaskComponent/README.md:33]: bg_task_before
bg_task_after

Comment (j0shuams) [reply src/Samples/BgTaskComponent/README.md:33]: these might be nice

**Review by j0shuams — APPROVED** [review]

File: src/Samples/BgTaskComponent/README.md [hunk 2]
```diff
@@ -2,19 +2,51 @@
...
 
-To build this sample, set **WpfApp.Package** as the startup project before building the solution. The background task raises a toast notification and is triggered by a time zone change. 
-
 This sample includes the following projects:
 
 - **BgTaskComponent**: This is a C#/WinRT component with an example background task that pops a toast notification.
+
 - **WpfApp** and **WpfApp.Package**: These projects demonstrate hosting the background task component in a packaged .NET 5 desktop (WPF) application.
+
   - **WpfApp** has a project reference to **BgTaskComponent**.
```
[the remark this answers is not in the document]
Comment (angelazhangmsft) [reply src/Samples/BgTaskComponent/README.md:11]: yes WpfApp does the UI, from my understanding it's the main app, but we need a packaging app in order to register the background task

File: src/Samples/BgTaskComponent/BgTaskComponent/BgTaskComponent.csproj [hunk 3]
```diff
@@ -13,7 +13,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.1.2-prerelease.210208.6" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.1.3" />
```
Comment (angelazhangmsft) [inline src/Samples/BgTaskComponent/BgTaskComponent/BgTaskComponent.csproj:16]: todo: update to 1.1.4

**Pull request merged**